backend/service/k8s/cronjob_test.go

```diff
@@ -106,7 +106,7 @@ func TestProtoForCron(t *testing.T) {
 					ConcurrencyPolicy:       v1beta1.AllowConcurrent,
 					Schedule:                "5 4 * * *",
 					Suspend:                 &[]bool{true}[0],
-					StartingDeadlineSeconds: &[]int64{69}[0],
+					StartingDeadlineSeconds: &[]int64{60}[0],
```

Comment:
Is this just fixing a typo?

Reply:
Yes, mcutalo said to stay away from "meme" value numbers like 69 and 420

Reply:
For this PR its fine, but going forward lets make a separate PR for fixes like these since its not related to the main task at hand